plugins/com.google.cloud.tools.eclipse.appengine.localserver/src/com/google/cloud/tools/eclipse/appengine/localserver/server/LocalAppEngineServerBehaviour.java

```diff
@@ -83,10 +82,12 @@ public final void setModulePublishState2(IModule[] module, int state) {
 
   /**
    * Starts the development server.
-   * @param runnables The path to directories that contain configuration files like appengine-web.xml
-   * @param stream
+   * @param runnables the path to configuration files and/or directories that contain configuration files 
+   * @param stream the stream to send development server process output to
+   * @param mode the mode of the server launch defined by ILaunchManager - RUN_MODE or DEBUG_MODE
+   * @param debugPort the port to attach a debugger to if launch is in debug mode
    */
-  void startDevServer(List<File> runnables, MessageConsoleStream stream) {
+  void startDevServer(List<File> runnables, MessageConsoleStream stream, String mode, int debugPort) {
```

Comment:
Removed the profile launch mode and added issue (#293) to decide if we want to support it and how.